[inferno-collection]/inferno-april-fools/server.lua

```diff
@@ -9,29 +10,43 @@
 --
 
 --
---		Nothing past this point needs to be edited, all the settings for the resource are found ABOVE this line.
+--		Nothing past this point needs to be edited, all the settings for the resource are found in the config.lua file.
 --		Do not make changes below this line unless you know what you are doing!
 --
 
 local AllPlayers = {}
+local players = {}
 
-RegisterServerEvent('April-Fools:Register')
-AddEventHandler('April-Fools:Register', function()
-    if not AllPlayers[source] then
-        AllPlayers[source] = true
-    end
-end)
+if Config.onConnect then
+    AddEventHandler("playerConnecting", function()
```

Comment:
No parameters defined, so an error is thrown on line 24.
```suggestion
    AddEventHandler("playerConnecting", function(_, _, deferrals)
```